modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/portlet/action/test/util/JSONObjectDefinitionTestUtil.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2024 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+package com.liferay.object.web.internal.portlet.action.test.util;
+
+import com.liferay.portal.kernel.json.JSONArray;
+import com.liferay.portal.kernel.json.JSONObject;
+
+import java.util.Objects;
+
+/**
+ * @author Eva Budai
+ */
+public class JSONObjectDefinitionTestUtil {
+
+	public static JSONObject getAuthorJSONObject(JSONObject jsonObject) {
+		return getObjectFieldJSONObject(jsonObject, "creator");
+	}
```

Comment:
I don't think we need specific methods for these, since we can just call the `getObjectFieldJSONObject()` method

Reply:
And by these i mean from line 18 to 38

Reply:
Plus, none of the methods here get properties that are inside the `objectFields` array